fake_switches/cisco/command_processor/config_interface.py

```diff
@@ -153,6 +166,19 @@ def do_no_ip(self, *args):
         if "redirects".startswith(args[0]):
             self.port.vendor_specific["no ip redirects"] = True
 
+        if "helper-address".startswith(args[0]):
+            if len(args) > 2:
+                self.write_line(" ^")
+                self.write_line("% Invalid input detected at '^' marker.")
+                self.write_line("")
+            else:
+                if len(args) == 1:
```

Comment:
Cisco supports removing all helpers but not brocade? (This is not a comment it is a legitimate question)